db/migrate/20211118000407_change_amount_to_be_decimal_in_groups.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAmountToBeDecimalInGroups < ActiveRecord::Migration[6.1]
+  def change
+    change_column :groups, :amount, :decimal
```

Comment:
I see the initial migration is taking care of this

Is this migration needed?

Reply:
we created this migration to change the value: integer of amount to decimal, if we are solved this change in local we should are not push this migration?

Reply:
i solve it

DONE